src/libPMacc/PMaccConfig.cmake

```diff
@@ -66,11 +66,19 @@ if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-std=c\\+\\+11")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
-set(CUDA_ARCH sm_20 CACHE STRING "Set GPU architecture")
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_10" IS_CUDA_ARCH_UNSUPPORTED)
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_11" IS_CUDA_ARCH_UNSUPPORTED)
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_12" IS_CUDA_ARCH_UNSUPPORTED)
-string(COMPARE EQUAL ${CUDA_ARCH} "sm_13" IS_CUDA_ARCH_UNSUPPORTED)
+set(CUDA_ARCH 20 CACHE STRING "Set GPU architecture (semicolon or comma separated list, e.g. '-DCUDA_ARCH=20,35,60')")
+
+# convert all commas to semicolons to support the following foreach call
+string(REPLACE "," ";" PMACC_CUDA_ARCH ${CUDA_ARCH})
+foreach(PMACC_CUDA_ARCH_ELEM ${PMACC_CUDA_ARCH})
+    # set flags to create device code for the given architecture
+    set(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS}
+        "--generate-code arch=compute_${PMACC_CUDA_ARCH_ELEM},code=sm_${PMACC_CUDA_ARCH_ELEM}")
+    string(COMPARE EQUAL ${PMACC_CUDA_ARCH_ELEM} "10" IS_CUDA_ARCH_UNSUPPORTED)
```

Comment:
make two checks:
- is the element an integer? no-> unsupported (without output of element string)
- is the integer >20? no-> unsupported (without output of element string)

this will help users to do the transition